inc/front/lazyload.php

```diff
@@ -77,8 +77,19 @@ class_loaded: "lazyloaded",
 </script>';
 
 	if ( get_rocket_option( 'lazyload_youtube' ) ) {
+		
+		/**
+		 * Filters the resolution of the YouTube thumbnail
+		 *
+		 * Accepted values: default, mqdefault, sddefault, hqdefault, maxresdefault
+		 *
+		 * @since 2.11.5
+		 * @author Arun Basil Lal
+		 */
```

Comment:
You're missing the @param here. As reference: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters